test/e2e/run_test.go

```diff
@@ -491,12 +486,18 @@ var _ = Describe("Podman run", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 
-		session = podmanTest.Podman([]string{"run", "--rm", "-e", "MYSQL_USER=zmuser", "-e", "MYSQL_PASSWORD=zmpass", "-e", "MYSQL_DATABASE=zm", "-e", "MYSQL_ROOT_PASSWORD=mysqlpassword", "docker.io/centos/mariadb-101-centos7", "ls", "-al", "/var/lib/mysql/data"})
+		dockerfile := `FROM busybox
+RUN mkdir -p /myvol/data && chown -R mail.0 /myvol
+VOLUME ["/myvol/data"]
+USER mail`
+
+		podmanTest.BuildImage(dockerfile, "test", "false")
+		session = podmanTest.Podman([]string{"run", "test", "ls", "-al", "/myvol/data"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
-		Expect(session.OutputToString()).To(ContainSubstring("mysql root"))
+		Expect(session.OutputToString()).To(ContainSubstring("mail root"))
```

Comment:
Should this be `test root`, not `mail root`?

Reply:
I am setting the user to `mail` in the dockerfile and chowning the directory with that as well, so it will be `mail root`. test is just the name I am giving to the image being build.